chia/cmds/chia.py

```diff
@@ -118,6 +119,7 @@ def run_daemon_cmd(ctx: click.Context, wait_for_unlock: bool) -> None:
 cli.add_command(stop_cmd)
 cli.add_command(netspace_cmd)
 cli.add_command(farm_cmd)
+cli.add_command(data_cmd)
 
```

Comment:
ok but seems that when we split to a different repo this should change and the main chia node should not include commands related to data-layer